docs/docs/components/guidelines/README.md

```diff
@@ -324,17 +356,42 @@ module.exports = {
 };
 ```
 
-##### Testing
+#### Testing
 
 Pipedream does not currently support unit tests to validate that changes to app
 files are backwards compatible with existing components. Therefore, if you make
 changes to an app file that may impact other sources, you must currently test
 potentially impacted components to confirm their functionality is not negatively
 affected. We expect to support a testing framework in the future.
 
+Some tips on what to test:
+
+- High traffic, or large record collections:
```

Comment:
Small feedback but can you make the top-level bullets here (the sections) `#####` tags, instead? That'll let us link to them as anchors and I think the section delineating reads easier than nested bullets.